connectors/src/connectors/slack/index.ts

```diff
@@ -594,6 +594,26 @@ export async function retrieveSlackContentNodes(
   return new Ok(contentNodes);
 }
 
+export async function getWhiteListedChannelPatterns(
+  connectorId: ModelId
+): Promise<Result<string, Error>> {
+  const slackConfig = await SlackConfigurationResource.fetchByConnectorId(
+    connectorId
+  );
+  if (!slackConfig) {
+    return new Err(
+      new Error(
+        `Failed to find a Slack configuration for connector ${connectorId}`
+      )
+    );
+  }
+  if (!slackConfig.whiteListedChannelPatterns) {
+    return new Ok("");
```

Comment:
As we discussed [here](https://github.com/dust-tt/dust/pull/5050#discussion_r1606550692), changing type to `null` requires to update `ConnectorConfigGetter` signature to return `Promise<Result<string | null, Error>>` instead of `Promise<Result<string, Error>>`.

Reply:
Will change it `null` anyways